CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+**6.0.1+1.11.1**
+
+- upgrade to Cilium v1.11.1
+- make sure Cilium Helm chart is installed before rendering (contribution by @tiagoblackcode)
```

Comment:
```suggestion
- make sure Cilium Helm repository is installed before rendering the template (contribution by @tiagoblackcode)
```

The check is for the repository, not the chart 🙂

Reply:
Doesn't matter. Nobody reads changelog's anyways :wink: I'll fix it with the next update.